strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/maven/MavenArtifactControllerTest.java

```diff
@@ -399,7 +401,37 @@ private void resolveArtifact(String artifactPath,
         assertEquals("MD5 checksums did not match!", md5Local, md5Remote);
         assertEquals("SHA-1 checksums did not match!", sha1Local, sha1Remote);
     }
+    
+    @Test
+    public void testHeadersFetch()
+            throws Exception
+    {   
+        String artifactPath;
+        Headers headersFromGET, headersFromHEAD;
+
+        /* Hosted Repository */
+        artifactPath = "storages/storage0/act-releases-1/org/carlspring/strongbox/browse/foo-bar/2.4/foo-bar-2.4.pom";
+        headersFromGET = client.getHeadersFromGET(artifactPath);
+        headersFromHEAD = client.getHeadersfromHEAD(artifactPath);
+        assertHeadersEquals(headersFromGET,headersFromHEAD);
+    }
+    
+    private void assertHeadersEquals(Headers h1, Headers h2)
+    {
+        assertNotNull(h1);
+        assertNotNull(h2);
+
+        //assertEquals(h1.size(),h2.size());
```

Comment:
//assertEquals(h1.size(),h2.size());

trash ?

Reply:
+1: @sanket407 , could you please remove all your newly-added commented out code, if it's not needed? Thanks! :)